sdk/servicebus/azure-servicebus/azure/servicebus/_common/message.py

```diff
@@ -128,6 +128,13 @@ def __init__(self, body, **kwargs):
     def __str__(self):
         return str(self.message)
 
+    def __repr__(self):
+        # type: () -> str
+        rec_repr = "body={}".format(
+            self.message._body # pylint: disable=protected-access
```

Comment:
self.body should be enough?
```suggestion
            self.body
```